apps/vmq_plugin/test/vmq_plugin_SUITE.erl

```diff
@@ -20,7 +20,6 @@ all() ->
     ].
 
 init_per_suite(Config) ->
-    application:ensure_all_started(lager),
     %%lager:set_loglevel(lager_console_backend, debug),
```

Comment:
```suggestion
    %%lager:set_loglevel(lager_console_backend, debug),
```

Really can remove the whole `init_per_suite/1` since it does nothing here